lib/route_translator/translator/path/segment.rb

```diff
@@ -16,17 +16,13 @@ def fallback_options(str, locale)
           def translate_string(str, locale, scope)
             locale = locale.to_s.gsub('native_', '')
             handler = proc { |exception| exception }
```

Comment:
is there any way to get rid of this `handler`?

Sorry but I don't have time to take a deep look at this